app/assets/stylesheets/lessons.scss

```diff
@@ -0,0 +1,727 @@
+// Place all the styles related to the Lessons controller here.
```

Comment:
Can you delete this entire file please? it's in this PR because master is merged in.